### DISCUSSION TITLE
how to get superchain faucet temporary access attestation

### DISCUSSION THREAD

**Discussion by sukrucildirr** [ethereum-optimism/developers#69] [General]
### Discussion Topic

how to get superchain faucet temporary access attestation to get much amount testnet token ?

**Comment by sbvegan** [comment]
Hey @gangstarr60, I've asked internally about the attestation. You can fill out the [bulk testnet form](https://docs.google.com/forms/d/e/1FAIpQLSfGURm08bQAnsy1B91uyGb3NHj5fLTXjtX-AUm9rlpS39YtJg/viewform) in the [faucet FAQ](https://app.optimism.io/faucet) and we can evaluate your request that way too.

**Reply by sukrucildirr** [reply]
thanks, i see now.

**Reply by sbvegan** [reply]
sure thing, the testnet ether has been distributed
